code/modules/food_and_drinks/kitchen_machinery/processor.dm

```diff
@@ -173,31 +176,27 @@
 	AM.pixel_y = -8 + (round(ii/3)*8)
 	return i
 
-/obj/machinery/processor/slime/process()
-	if(processing)
-		return
-	var/mob/living/simple_animal/slime/picked_slime
-	for(var/mob/living/simple_animal/slime/slime in ohearers(1,src))
-		if(slime.stat)
-			picked_slime = slime
-			break
-	if(!picked_slime)
-		return
-	var/datum/food_processor_process/P = select_recipe(picked_slime)
-	if (!P)
-		return
-
-	visible_message("[picked_slime] is sucked into [src].")
-	picked_slime.forceMove(src)
+/obj/machinery/processor/slime/interact(mob/user)
+	. = ..()
+	if(sbacklogged)
+		for(var/mob/living/simple_animal/slime/AM in hearers(1,src)) //fallback in case slimes got placed while processor was active
```

Comment:
```suggestion
		for(var/mob/living/simple_animal/slime/AM in ohearers(1,src)) //fallback in case slimes got placed while processor was active
```

Reply:
Also this would move all of the adjacent slimes, defeating the point of `processing`

Reply:
reason of why i use hearers here instead of ohearers is cause hearers includes the center too i know from playing xenobio that many players usually place slimes directly ontop of the processor

Reply:
Also this is simply there to act as an fallback if someone places down slimes during the time where it processes because the proximity monitor won't trigger for the slimes again except if someone moves them

Reply:
after it finishes it simply then pulls in all the slimes adjacent to it yes trough that will only trigger after the interact proc of the parent is finished and at that time processing will be set to false again

Reply:
its basically just a way to recheck for all the slimes that got put down while the processor was active afterwards beacause the proximity monitor can't do that so those won't just keep sitting there